### DISCUSSION TITLE
Deleting Nonfuctional Domains/Accounts

### DISCUSSION THREAD

**Discussion by PierceWHoward** [ThioJoe/YT-Spam-Lists#70] [General]
Hi,

I was hoping to get some clarification on why nonfunctional domains or accounts are removed from the database. I know the reasoning behind removing them is that they can no longer scam people but regardless viewers and creators don't want to see them on videos. Old videos may still retain these comments and have to be deleted, plus it would be useful to report for YouTube to identify old spam to remove new spam. Is there another reason for the removal of these items?

Thanks!

**Comment by KendallDoesCoding** [comment]
@ThioJoe I removed only 2 domains not thinking about this, will revert changes if required.

**Reply by KendallDoesCoding** [reply]
#81
